cypress/integration/examples/querying.spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
+
+context('Querying', () => {
+  beforeEach(() => {
```

Comment:
The test suite should be testing the Book Store app functionality as described in the task, not querying examples from Cypress documentation.